src/app/views/marketplace/marketplace-app.html

```diff
@@ -36,10 +37,18 @@ <h2 class="media-heading">{{vm.app.name}}</h2>
             <div class="well text-center well-details">
               <div class="row">
                 <div class="col-xs-12 col-sm-4 col-sm-offset-4 col-md-8 col-md-offset-2">
-                  <div class="btn-fluid-wrapper">
-                    <a href="" ng-click="vm.provisionLink()" class="btn btn-warning btn-promo">
+                  <div class="btn-fluid-wrapper" ng-switch on="vm.appInstallationStatus()">
+                    <div ng-switch-when="CONFLICT" class="message">
+                      <i class="fa fa-exclamation-triangle"></i>
+                      {{ 'mno_enterprise.templates.dashboard.marketplace.show.conflicting_app' | translate }}
+                      <a ui-sref="home.marketplace.app({appId: vm.conflictingAppInstance.app_id})">{{vm.conflictingAppInstance.app_name}}</a>
```

Comment:
I would, but how do I use interpoloate params, if I want to put an html element, and not only text ?